hack/sync.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# This script sync CRDs from upstream prometheus-operator helm chart
+
+version=${VERSION:-main}
+tmp="/tmp/helm-charts-$version"
+upstream=https://github.com/prometheus-community/helm-charts
+
+
+function main() {
+git clone --depth 1  --branch "kube-prometheus-stack-${version}" "$upstream" "$tmp"
+rm -rf ./helm/prometheus-operator-crd/templates/
```

Comment:
This allow to run the script from anywhere
```suggestion
rm -rf "$script_dir/../helm/prometheus-operator-crd/templates/
```